Docs: add note about ability to build experimental snaps from true forks (#35)
FreeCAD/FreeCAD-snap: docs/index.md

There is an option to build Snaps from experimental commits, branches, or even git release tags. 

> **Note:** it is possible to build experimental builds from true forks of FreeCAD master branch ([ref](https://github.com/FreeCAD/FreeCAD/pull/7118#issuecomment-1171458436))

Reviewer: ```suggestion
**Note:** It is possible to create experimental builds from any true fork of the upstream FreeCAD repository ([ref](https://github.com/FreeCAD/FreeCAD/pull/7118#issuecomment-1171458436))
```
